src/utils/status/pv/constants.js

```diff
@@ -0,0 +1,8 @@
+export const PV_STATUS_AVAILABLE = 'PV_STATUS_AVAILABLE';
+export const PV_STATUS_BOUND = 'PV_STATUS_BOUND';
+export const PV_STATUS_RELEASED = 'PV_STATUS_RELEASED';
+export const PV_STATUS_FAILED = 'PV_STATUS_FAILED';
+export const PV_STATUS_DEFAULT = 'PV_STATUS_DEFAULT';
+
+export const PV_STATUS_ALL_ERROR = [PV_STATUS_FAILED];
+export const PV_STATUS_ALL_PROGRESS = [PV_STATUS_AVAILABLE];
```

Comment:
does it make sense to have Available PV shown as inProgress?

Reply:
@suomiy The PV that are not bound and are available to be claimed I put them under this category. I was also bit confused about it, if you can provide some feedback on what should be shown in this category.